webdriver-spec.html

```diff
@@ -5128,12 +5128,12 @@
 
   <li><p>Let <var>i</var> be 0.
 
-  <li><p>Let <var>length</var> be the length of <var>character array</var>.
+  <li><p>Let <var>length</var> be the length of <var>text</var>.
 
   <li><p>While <var>i</var> &lt; <var>length</var>:
 
     <ol>
-      <li><p>Let <var>char</var> be the entry in <var>character array</var> at
+      <li><p>Let <var>char</var> be the entry in <var>text</var> at
         index <var>i</var>.
```

Comment:
This would iterate by character and not code point. I think we need to decide if we want to iterate on code points or grapheme clusters, and right now I tend to lean towards ignoring graphemes for the time being.

Can you change this to iterate without an index, like this?

` <li><p>For each <a><var>code point</var></a> in <var>text</var>:`